web-components/src/components/cards/CredibilityCard/CredibilityCard.tsx

```diff
@@ -0,0 +1,62 @@
+import { Box, useMediaQuery, useTheme } from '@mui/material';
+
+import { BlockContent } from '../../block-content';
+import OutlinedCheckIcon from '../../icons/OutlinedCheckIcon';
+import { Body, Title } from '../../typography';
+import Card from '../Card';
+import { CredibilityCardProps } from './CredibilityCard.types';
+
+export const CredibilityCard: React.FC<CredibilityCardProps> = ({
+  index = 0,
+  title,
+  descriptionRaw,
+  icon,
+  claims,
+}) => {
+  const theme = useTheme();
+  const lg = useMediaQuery(theme.breakpoints.up('lg'));
+  const background = `${Math.min(100, 55 + index * (lg ? 12 : 8))}%`;
```

Comment:
Could we do the same thing without the client media query check? This is to start making components that are SSR-friendly.

Something like that:
```
const backgroundMobile = `${Math.min(100, 55 + index * 8)}%`;
const backgroundDesktop = `${Math.min(100, 55 + index * 12)}%`;

background: {xs: `linear-gradient(-8deg, #F1F7F6 ${backgroundMobile}, transparent ${backgroundMobile})`, lg: `linear-gradient(-8deg, #F1F7F6 ${backgroundDesktop}, transparent ${backgroundDesktop})`}
```

Reply:
good point, I added some util fn to make the code more readable